Library/Homebrew/dev-cmd/dispatch-build-bottle.rb

```diff
@@ -25,53 +25,62 @@ def dispatch_build_bottle_args
              description: "Dispatch specified workflow (default: `dispatch-build-bottle.yml`)."
       switch "--upload",
              description: "Upload built bottles to Bintray."
+      switch "--linux",
+             description: "Dispatch bottle for Linux (using GitHub runners)."
 
+      conflicts "--macos", "--linux"
       named_args :formula, min: 1
     end
   end
 
   def dispatch_build_bottle
     args = dispatch_build_bottle_args.parse
 
-    # Fixup version for ARM/Apple Silicon
-    # TODO: fix label name to be 11-arm64 instead and remove this.
-    args.macos&.gsub!(/^11-arm$/, "11-arm64")
+    tap = Tap.fetch(args.tap || CoreTap.instance.name)
+    user, repo = tap.full_name.split("/")
+    ref = "master"
+    workflow = args.workflow || "dispatch-build-bottle.yml"
 
-    macos = args.macos&.yield_self do |s|
-      MacOS::Version.from_symbol(s.to_sym)
-    rescue MacOSVersionError
-      MacOS::Version.new(s)
-    end
+    if args.macos
+      # Fixup version for ARM/Apple Silicon
+      # TODO: fix label name to be 11-arm64 instead and remove this.
+      args.macos&.gsub!(/^11-arm$/, "11-arm64")
+
+      macos = args.macos&.yield_self do |s|
```

Comment:
```suggestion
      args.macos.gsub!(/^11-arm$/, "11-arm64")

      macos = args.macos.yield_self do |s|
```

since I think we guarantee non-nil here.

Reply:
Good catch, done.

Reply:
`brew typecheck` complains about it tho.

Reply:
Extract to a local variable maybe, e.g. `if (macos = args.macos)`

Reply:
That worked, thanks.